crap-advisor-frontend/src/store/reviews/saga/index.ts

```diff
@@ -0,0 +1,59 @@
+import { Dispatch } from 'react';
+
+import { PayloadAction } from '@reduxjs/toolkit';
+import { AxiosError, AxiosResponse } from 'axios';
+import { SagaIterator } from 'redux-saga';
+import {
+  StrictEffect,
+  all,
+  call,
+  put,
+  select,
+  take,
+  takeLatest,
+} from 'redux-saga/effects';
+
+import { Api } from 'app/api';
+import { ApiEndpoint } from 'app/constants';
+import { IReview, Restroom } from 'app/typings';
+
+import { success as restroomSuccess } from 'store/restrooms/slice';
+import { error, setCurrentRestroom, start, success } from 'store/reviews/slice';
+
+function* getReviewsByIdSaga(
```

Comment:
wtf counter: 99999

Reply:
Damn, hope you not lost your mind, because i can't see counter and this magic damn number.